DOCSP-13820 explain method (#92)
mongodb/docs-java: source/fundamentals/aggregation.txt

   {"_id": 4, "count": 2}
   {"_id": 5, "count": 1}

Explain Aggregation Example
~~~~~~~~~~~~~~~~~~~~~~~~~~~

> To explain the execution plan of the operation, append the :java-docs:`explain()

Reviewer: ```suggestion
To explain the execution plan of the operation, call the :java-docs:`explain()
```

Issue: I think this should say "call the explain method". I don't believe you can "append a method" (you can call a method on the item returned from another method in a method chain, but I believe appending as a verb is reserved for adding items to the end of array-like data structures). 

I think this paragraph should be rephrased such that it is focused on the ``FindIterable`` object returned from  the call to the ``MongoCollection.aggregate()`` method rather than the `explain()` method appended to `aggregate()`.

<apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html#explain()>`

> method containing a :java-core-api:`verbosity level <com/mongodb/ExplainVerbosity>` to the ``aggregates()`` method.

Reviewer: ```suggestion
method of the ``FindIterable`` class containing a :java-core-api:`verbosity level <com/mongodb/ExplainVerbosity>` to the ``aggregates()`` method.
```

Issue (possibly suggestion): Could the class the method belongs to be stated? I don't remember if we agreed on this as a team, but it feels jarring to have a method stated without the class it belongs to.

> method containing a :java-core-api:`verbosity level <com/mongodb/ExplainVerbosity>` to the ``aggregates()`` method.

Reviewer: Issue: I don't think it is correct to say "containing a verbosity level" to describe the verbosity level passed to the method. I think this should be rephrased to be closer to "pass the explain() method a verbosity level to determine the contents of the explanation".

> The following table shows the verbosity levels you can give an
> operation, what they do, and what their intended use case is:

Reviewer: ```suggestion
The following table shows the verbosity levels for explanations
and their intended use cases:
```

Issue: I do not believe a verbosity level can be given to any operation. I believe it is only given to an explain operation.

Author: Reworded to:
```suggestion
The following table shows the verbosity levels for the explain
operation and their intended use cases:
```
